CodeGeneration/Sources/SyntaxSupport/SyntaxNodeKind.swift

```diff
@@ -378,7 +378,7 @@ public enum SyntaxNodeKind: String, CaseIterable {
     case .closureCapture: return "closureCaptureItem"
     case .closureCaptureClause: return "closureCaptureSignature"
     case .closureCaptureList: return "closureCaptureItemList"
-    case .closureCaptureSpecifier: return "closureCaptureItemSpecifier"
+    case .closureCaptureModifier: return "closureCaptureItemSpecifier"
```

Comment:
I don't have a strong opinion on modifier vs specifier here. But seems weird to have both still.